.github/workflows/ci.yaml

```diff
@@ -194,6 +194,28 @@ jobs:
             exit 1
           fi
 
+  python-package:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: ${{ env.PYTHON_VERSION }}
+          architecture: x64
+      - name: "Prep README.md"
+        run: python scripts/transform_readme.py --target pypi
+      - name: "Build wheels"
+        uses: PyO3/maturin-action@v1
+        with:
+          manylinux: auto
```

Comment:
We may want to use `container: off` to run this directly on the host instead of a container (for speed?) this is closer to the release build though so 🤷‍♀️ 

https://github.com/PyO3/maturin-action